test/integration/query-interface.test.js

```diff
@@ -336,7 +336,8 @@ describe(Support.getTestDialectTeaser('QueryInterface'), () => {
     it('rename a simple column', function() {
       const self = this;
       const Users = self.sequelize.define('_Users', {
-        username: DataTypes.STRING
+        username: DataTypes.STRING,
+        commit: DataTypes.STRING
```

Comment:
@felixfbecker This is testing that the column names are properly quoted. Since `commit` is a reserved word in SQL, any query that is run using that as a column name will fail, unless that column name is quoted.

```
SequelizeDatabaseError: SQLITE_ERROR: near "commit": syntax error
```

See the bug I filed for this issue: https://github.com/sequelize/sequelize/issues/7758